Lib/importlib/_bootstrap_external.py

```diff
@@ -767,10 +791,29 @@ def get_code(self, fullname):
                 except OSError:
                     pass
                 else:
+                    exc_details = {
+                        'name': fullname,
+                        'path': bytecode_path,
+                    }
                     try:
-                        bytes_data = _validate_bytecode_header(data,
-                                source_stats=st, name=fullname,
-                                path=bytecode_path)
+                        flags = _classify_pyc(data, fullname, exc_details)
+                        bytes_data = data[16:]
```

Comment:
Consider using `memoryview(data)[16:]` to avoid a large copy assuming _compile_bytecode is happy with that (it appears to be, `marshal.loads()` accepts `memoryview`)

Reply:
sgtm